Lecture-6/Task1/l6task1/ProductGenerator.java

```diff
@@ -0,0 +1,105 @@
+package l6task1;
+
+import l6task1.factories.CoffeeFactory;
+import l6task1.factories.PackFactory;
+import l6task1.factories.ProductFactory;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class ProductGenerator {
+    private static final String INPUT_FILE = "c:/Work folder/Automation/Lesson#8/INPUT_FILE.txt";
```

Comment:
Never use absolute paths. Place file into `resources` folder in you project and use relative path like `./INPUT_FILE.txt`